packages/openssl/packaging

```diff
@@ -0,0 +1,11 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+mkdir ${BOSH_INSTALL_TARGET}/openssl
+echo "Extracting openssl"
+tar xzf openssl/openssl-1.0.2j.tar.gz
```

Comment:
We should probably consider adding a small documentation (in readme.md or in a new docs/keepalived.md file) explaining how to use keepalived for ha (e.g. with sample bosh manifest extract for a simple pair of haproxy vms in a single az, or a new bosh manifest template), and how to test the automatic fail over (e.g. bosh stop on one of the node, traces generated on the other node) that would be useful for monitoring a fail over